core/arch/arm/plat-sam/conf.mk

```diff
@@ -45,6 +57,14 @@ $(call force,CFG_CORE_DYN_SHM,n)
 
 CFG_MMAP_REGIONS ?= 24
 
+ifeq ($(CFG_SAMA7G5),y)
+CFG_SHMEM_START  = 0x61000000
```

Comment:
use either `CFG_xxx ?= <VALUE>` or `$(call force,CFG_xxx,<VALUE>)` for `CFG_` config switches

Reply:
Updated, thanks.